addons/mail/static/src/models/chat_window/chat_window.js

```diff
@@ -91,8 +94,11 @@ function factory(dependencies) {
          * @param {Object} [param0={}]
          * @param {boolean} [param0.notifyServer=true]
```

Comment:
Don't forget to update docstring of all of these methods according to change of default

Reply:
Done

Reply:
All of those docstring still state that true is the default value.

Reply:
Done